Source/Dafny/Resolver.cs

```diff
@@ -4314,8 +4314,8 @@ public class ModuleBindings {
 
       // Finally, of the big-enough native types, pick the first one that is
       // supported by the selected target compiler.
-      foreach (var nativeT in bigEnoughNativeTypes) {
-        if ((nativeT.CompilationTargets & DafnyOptions.O.CompileTarget) != 0) {
+      foreach (var nativeT in bigEnoughNativeTypes) { // FIXME shouldn't the resolver be agnostic to the compilation target?
```

Comment:
FIXME. I suggest you cut an issue.

Reply:
https://github.com/dafny-lang/dafny/issues/1732